settings

```diff
@@ -1,24 +1,34 @@
-VERSION="1.23"
+FOREMAN_VERSION="1.23"
+KATELLO_VERSION="3.13"
```

Comment:
Should this be part of foreman's `SETTINGS` since they're quite tied together? Especially since we've started to share GPG keys now.

Reply:
not sure what you mean here

Reply:
We currently have `releases/foreman/x.y/settings` and `releases/katello/a.b/settings`. However, could we have a `KATELLO_VERSION` in `releases/x.y/settings` and account for this in the scripts? Longer term I'd prefer for katello tags to move to `foreman-katello-x.y-$os` rather than `katello-a.b-$os` which would remove a lot of the specifics again.

This doesn't have to be done in this PR, but I do want to get your thoughts on the direction.